app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
```

Comment:
- [ ] Please kindly add association between the user and post also, for example, the user should also have many post associated with the user foreign key

Reply:
hello @DanSam5K 
      I think you are in the wrong PR this is for the view, not the association ^^
and this is my second time doing this and the previous reviewers accepted the code so I don't understand what is the problem?

[the previous PR](https://github.com/yahyaelganyni1/the-daily-blog/pull/4)

Reply:
Hi @yahyaelganyni1,
I checked and the files present for me to review was just a single file which did not reflect your work and I saw you had two PR, with the task contain in the other PR so I reviewed the work on it.